spec/appliance_console/database_replication_standby_spec.rb

```diff
@@ -306,27 +306,44 @@
       double("nil PG::Result", :map_types! => mapped_result)
     end
 
-    before do
-      expect(PG::Connection).to receive(:new).and_return(connection)
-      expect(PG::BasicTypeMapForResults).to receive(:new).and_return(type_map)
-      subject.node_number = node_number
-    end
+    context "connection successful and node_number got" do
+      before do
+        expect(PG::Connection).to receive(:new).and_return(connection)
+        expect(PG::BasicTypeMapForResults).to receive(:new).and_return(type_map)
+        subject.node_number = node_number
+      end
 
-    it "returns true if no node is found" do
-      expect(connection).to receive(:exec_params).with(instance_of(String), [1]).and_return(nil_result)
-      expect(subject.node_number_valid?).to be_truthy
-    end
+      it "returns true if no node is found" do
+        expect(connection).to receive(:exec_params).with(instance_of(String), [1]).and_return(nil_result)
+        expect(subject.node_number_valid?).to be_truthy
+      end
 
-    it "returns true if a node is found and overwrite is confirmed" do
-      expect(connection).to receive(:exec_params).with(instance_of(String), [1]).and_return(one_result)
-      expect(subject).to receive(:ask_yn?).and_return(true)
-      expect(subject.node_number_valid?).to be_truthy
+      it "returns true if a node is found and overwrite is confirmed" do
+        expect(connection).to receive(:exec_params).with(instance_of(String), [1]).and_return(one_result)
+        expect(subject).to receive(:ask_yn?).and_return(true)
+        expect(subject.node_number_valid?).to be_truthy
+      end
+
+      it "returns false if a node is found and overwrite is not confirmed" do
+        expect(connection).to receive(:exec_params).with(instance_of(String), [1]).and_return(one_result)
+        expect(subject).to receive(:ask_yn?).and_return(false)
+        expect(subject.node_number_valid?).to be_falsey
+      end
     end
 
-    it "returns false if a node is found and overwrite is not confirmed" do
-      expect(connection).to receive(:exec_params).with(instance_of(String), [1]).and_return(one_result)
-      expect(subject).to receive(:ask_yn?).and_return(false)
-      expect(subject.node_number_valid?).to be_falsey
+    context "unable to get node_number" do
+      it "error to connect database" do
```

Comment:
These specs should describe expected behavior so this should probably be something like:

```ruby
it "returns false when the connection fails" do
```